chia/_tests/wallet/sync/test_wallet_sync.py

```diff
@@ -189,9 +190,6 @@ async def test_basic_sync_wallet(
 
     await add_blocks_in_batches(blocks_reorg[1:], full_node, blocks_reorg[0].header_hash)
 
-    for wallet_node, wallet_server in wallets:
-        await disconnect_all_and_reconnect(wallet_server, full_node_server, self_hostname)
-
```

Comment:
we dont need the reconnect now for the reorg to work, now it makes the wallet fail the sync since the peer disconnects from the wallet